ross/tests/test_rotor_assembly.py

```diff
@@ -738,25 +738,37 @@ def test_static_analysis_rotor3(rotor3):
         decimal=6,
     )
     assert_almost_equal(
-        rotor3.Vx,
-        [
-            0,
-            -494.2745,
-            -456.6791,
-            -419.0836,
-            -99.4925,
-            -61.8971,
-            -24.3016,
-            480.9807,
-            518.5762,
-            556.1716,
-            0,
-        ],
+        rotor3.Vx[0],
+        np.array(
+            [
+                -3.58824082e-12,
+                -4.94274533e02,
+                -4.56679111e02,
+                -4.19083689e02,
+                -9.94925251e01,
+                -6.18971030e01,
+                -2.43016808e01,
+                4.80980792e02,
+                5.18576214e02,
+                5.56171636e02,
+                -3.58824082e-12,
+            ]
+        ),
```

Comment:
I'm not quite sure I understand these changes relevance, since the decimals parameter is set to 3, both arrays will output exactly the same results when using assert_almost_equal method.